qemu/tests/migration_multi_host.py

```diff
@@ -48,6 +55,17 @@ def run(test, params, env):
                           % pre_sub_test, logging.info)
             utils_test.run_virt_sub_test(test, params, env, pre_sub_test)
 
+        logging.info("Try to login guest before migration test...")
+        vm = env.get_vm(vms[0])
+        if vm is None:
+            env_process.preprocess_vm(test, params, env, vms[0])
+            vm = env.get_vm(vms[0])
+
+        session = vm.wait_for_login(timeout=login_timeout)
+        logging.info("Start the background command in the guest...")
+        session.sendline(migration_bg_command)
+        time.sleep(5)
```

Comment:
Another note: We have some tests that don't want to login into guest at the first stage. So please make this configurable when you add the code.